openedx/features/enterprise_support/api.py

```diff
@@ -241,6 +243,10 @@ def enterprise_customer_for_request(request, tpa_hint=None):
 
     ec = None
 
+    running_pipeline = get_partial_pipeline(request)
```

Comment:
After a little digging, I was able to understand why you did this. I think it's worth adding comments for each of the ways we try to determine the enterprise customer for the request.